examples/cotire/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2017, Pawel Bylica
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-cotire)
+
+# download cotire
+hunter_add_package(cotire)
+message(${COTIRE_ROOT})
+
+# This works, but is quite ugly. Can we get include(cotire) to work?
```

Comment:
I don't think creating a hunterized fork of this project make sense, unless they are going to pick it up upstream.

Would this work if the cotire had a simple CMakeLists.txt with install() that installs the cotire.cmake module?